crates/wash-lib/src/build/component.rs

```diff
@@ -307,17 +318,16 @@ async fn build_tinygo_component(
             let mut args = vec![
             "build",
             "-o",
-            filename.as_str(),
+            &output_file_path,
             "-target",
             tinygo_config.build_target(&component_config.wasm_target),
             "-wit-package",
-            wit_directory.to_str().context("missing wit directory")?,
+            common_config.wit_path.to_str().context("missing wit directory")?,
```

Comment:
```suggestion
            common_config.wit_path.to_str().context("missing wit directory (`wit_path` not specified)")?,
```
Feels like we should check this higher up/make sure it's there earlier but this is also fine

Reply:
Oh yeah this is actually the error that the wit_path couldn't be converted to a str because it's invalid unicode, I'll indicate that